src/app/route/index.js

```diff
@@ -220,6 +232,24 @@ class AppRouter extends PureComponent {
         );
     }
 
+    renderMainItems() {
+        const { isOffline, isBig } = this.props;
+
+        if (isOffline && isBig) {
+            return <OfflineNotice isPage />;
```

Comment:
Maybe let's render it the same place as the error notice? Is it possible?

Reply:
It is possible, but we will be required to copy rendering of header and footer. Should I move it then?